### PR TITLE
Api/test

### PR DESCRIPTION
**Pull request by ovekyc**
Move request parse code to controller 
(we consider about making models for data handling)

Add delete an item code

### CODE REVIEW

File: src/server/items/controllers.js [hunk 1]
```diff
@@ -1,18 +1,34 @@
 import db from '../database';
 
 export default {
-  getAll: (res, cb) => {
-    const result = {};
+  getAll: (req, res, cb) => {
+    let result = {};
```
Comment (fguy) [inline src/server/items/controllers.js]: Why did you change this variable modifier back to "let"?

Comment (ovekyc) [reply src/server/items/controllers.js]: Oh it was my mistake 
I corrected it and pushed again

**Comment by fguy** [comment]
LGTM
